bugbug/rust_code_analysis_server.py

```diff
@@ -50,7 +56,7 @@ def start_process(self, thread_num: Optional[int] = None):
                 cmd += ["-j", str(thread_num)]
             self.proc = subprocess.Popen(cmd)
         except FileNotFoundError:
-            raise Exception("rust-code-analysis is required for code analysis")
+            raise AnalysisException("rust-code-analysis is required for code analysis")
```

Comment:
Still unresolved!

Reply:
Do you want me to remove that class? If I remove it, the exceptions are showing not defined error.

Reply:
not if we use `RuntimeError` instead

Reply:
We got some misunderstanding here. I meant to drop the `AnalysisException` class that you added, not the one that already exists in the code.

Reply:
I didn't add any extra class, I have added the class back.

Reply:
You add [this one](https://github.com/mozilla/bugbug/pull/3339/files#diff-fcf133d306cfeeb4178681c4fb3278231f3719ee2d12235a53620f8d16211909R22-R27)

Reply:
Oh okay.

Reply:
This is code review feedback, not instructions. Let us understand the goals here. 

You added the `AnalysisException` class in the `repository.py` file.  I suggested dropping it and using `RuntimeError` instead since it is not an analysis error, and we already have a class with that name in another file.

You should not replace `AnalysisException` in the `repository.py` file. The feedback is related to the `rust_code_analysis_server.py` file.